src/components/common/CourtOrderPoa.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <section id="court-order-poa">
+    <header>
+      <h2>{{sectionNumber}} Court Order and Plan of Arrangement</h2>
+      <div class="py-4">
+        If this filing is pursuant to a court order, enter the court order number. If this
+        filing is pursuant to a plan of arrangement, enter the court order number and select
+        Plan of Arrangement.
+      </div>
+    </header>
+    <div :class="{ 'invalid-section': invalidCourtOrder }">
+      <CourtOrderPoaShared
+        id="court-order"
+        :autoValidation="getAppValidate"
+        :hasDraftPlanOfArrangement="hasDraftPlanOfArrangement"
+        :invalidSection="invalidCourtOrder"
+        @emitCourtNumber="setFileNumber($event)"
+        @emitPoa="setHasPlanOfArrangement($event)"
+        @emitValid="setValidCourtOrder($event)"
+      />
+    </div>
+  </section>
+</template>
+
+<script lang="ts">
+import { Action, Getter } from 'vuex-class'
+import { Component, Prop, Vue } from 'vue-property-decorator'
+import { CourtOrderPoa as CourtOrderPoaShared } from '@bcrs-shared-components/court-order-poa'
+import { ActionBindingIF, FlagsReviewCertifyIF } from '@/interfaces/'
+
+@Component({
+  components: {
+    CourtOrderPoaShared
+  }
+})
+export default class CourtOrderPoa extends Vue {
+  /** Prop to provide section number. */
+  @Prop({ default: '' }) readonly sectionNumber: string
+
+  /** Store getters */
+  @Getter getFlagsReviewCertify!: FlagsReviewCertifyIF
+  @Getter getAppValidate!: boolean
+  @Getter hasDraftPlanOfArrangement!: boolean
+
+  /** Global actions */
+  @Action setValidCourtOrder!: ActionBindingIF
+
+  /** LocalProperties */
+  private courtOrderNumber: string = null
+  private planOfArrangement: boolean = false
+
+  /** Flags to enable validation */
+  private setFileNumber (courtOrderNumber: string): void {
+    this.courtOrderNumber = courtOrderNumber
+  }
+
+  private setHasPlanOfArrangement (planOfArrangement: boolean): void {
+    this.planOfArrangement = planOfArrangement
```

Comment:
This saves the event value in this component, but not in the store. Did you mean to call action `setHasPlanOfArrangement`?

Reply:
So saving the event value in the component would be lost after you navigate away from the page and instead using `@Action setHasPlanOfArrangement` would instead change the object in store?

Reply:
Yes.

To be precise, the value in the component will continue to exist until the component is destroyed.

We save data in the Vuex Store so it survives component creation/destruction, but also because it's global and convenient. (The store is lost if the user navigates away from this web app. But the app may support several routes (eg, steps in Create UI) and in that case the app is still loaded and running and everything is fine.)

Notably, the code that builds the filing for saving to the API (in filing-template-mixin.ts) gets the data from the store.